test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -0,0 +1,103 @@
+from packaging.version import Version
+
+import pytest
+
+from invoke.context import Context
+
+from test import test_utils
+
+UTILITY_PACKAGES_IMPORT = [
+    "bokeh",
+    "imageio",
+    "plotly",
+    "seaborn",
+    "shap",
+    "sagemaker",
+    "pandas",
+    "cv2"
+]
+
+# TODO: Need to be added to all DLC images in furture.
+@pytest.mark.model("N/A")
+@pytest.mark.integration("awscli")
+def test_awscli(mxnet_inference):
+    """
+    Ensure that boto3 is installed on mxnet inference
+
+    :param mxnet_inference: ECR image URI
+    """
+    image = mxnet_inference
+    ctx = Context()
+    container_name = test_utils.get_container_name("awscli", image)
+    test_utils.start_container(container_name, image, ctx)
+
+    test_utils.run_cmd_on_container(container_name, ctx, "which aws")
+    test_utils.run_cmd_on_container(container_name, ctx, "aws --version")
+
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("bokeh")
+def test_utility_packages_using_import(training):
+    """
+    Verify that bokeh is installed in the Training DLC image
+
+    :param training: training ECR image URI
+    """
+    ctx = Context()
+    container_name = test_utils.get_container_name("utility_packages_using_import", training)
+    test_utils.start_container(container_name, training, ctx)
+
+    framework, image_framework_version = test_utils.get_framework_and_version_from_tag(training)
+    if "mxnet" in framework:
+        if Version(image_framework_version) <= Version("1.8.0"):
+            pytest.skip("Extra utility packages will be added going forward")
```

Comment:
Please make this a dictionary for reference, rather than 3 different if conditions, for readability.

Example:

```
utility_package_minimum_framework_version = {
    "mxnet": "1.8",
    "pytorch": "1.7",
    "tensorflow2": "2.4",
    "tensorflow1": "1.15",
}
framework, framework_version = get_framework_and_version_from_tag(image)
framework = "tensorflow1" if framework == "tensorflow" and framework_version.startswith("1.") else "tensorflow2"
if Version(framework_version) < Version(utility_package_minimum_framework_version[framework]):
    pytest.skip("Extra utility packages will be added going forward.")
```